cypress/e2e/NumVerify/pricing.cy.js

```diff
@@ -0,0 +1,128 @@
+        cy.componentVisiblityCheck(':nth-child(4) > .inside > [data-tip="Detect carrier and line type information,<br> decide which communications to use, <br>increase leads and fight fraud."]',  '\n                        Carrier & Line TypeDetection\n                        ');
+        cy.componentVisiblityCheck(':nth-child(4) > .inside > [data-tip="Know the geographical information behind any<br> phone number, for many numbers even down <br>to the city of registration."]', '\n                        Country & LocationDetection\n                        ');
+        cy.componentVisiblityCheck(':nth-child(4) > .inside > [data-tip="Paid customers may access the API and<br> all validation data securely via 256-bit<br> HTTPS Encryption (SSL)."]', '\n                        256-bit HTTPSEncryption\n                        ');
+    });
+
+    it('16. Toggle the "Subscription" button to yearly', () => {
+        cy.get('.yearly_price').click();
+    });
+ 
+    it('17.Test to check the "Free plan" price and text', () => {
+        cy.componentVisiblityCheck(':nth-child(1) > .inside > header',  '\n              \t\t\t\tFree Plan\n                        ');
+        cy.componentVisiblityCheck(':nth-child(1) > .inside > .pricing > .yearly_price_data', '\n                            $\n                        \t0.00\n                        \t/ yearly\n                        ');
+    });
+ 
+    it('18.Test to check the "Basic plan" price and text', () => {
+        cy.componentVisiblityCheck('header.border_right_under_700', 'Basic');
+        cy.componentVisiblityCheck(':nth-child(2) > .inside > .pricing > :nth-child(2)',  '\n                            $\n                        \t161.99\n                        \t/ yearly\n                            \n                        ');
+    });
+
+    it('19.Test to check the "Professional Plan" price and text', () => {
+        cy.componentVisiblityCheck('header.highlighted', 'Professional Plan'); 
+        cy.componentVisiblityCheck('td.highlighted > .inside > .pricing > :nth-child(2)',   '\n                            $\n                        \t629.99\n                        \t/ yearly\n                            \n                        ');
+    });
+ 
+    it('20.Test to check the "Enterprise Plan" price and text', () => {
+        cy.componentVisiblityCheck('.no_border_left', 'Enterprise Plan');
+        cy.componentVisiblityCheck(':nth-child(4) > .inside > .pricing > :nth-child(2)',  '\n                            $\n                        \t1,325.99\n                        \t/ yearly\n                            \n                        ');
+    });
+
+    it('21.Test to check the "Volume" price and text', () => {
+        cy.componentVisiblityCheck('.inside > .pricing > p', 'Volume');
+        cy.componentVisiblityCheck(':nth-child(5) > .inside > header',  'Looking for more?Contact us for a quote .');
+    });
+});
```

Comment:
add test case for button click of find out more and check the navigation happened or not